### DISCUSSION TITLE
Feature Request: Improve reporting of error statuses within manage hostname

### DISCUSSION THREAD

**Discussion by lakesol** [umbraco/Umbraco.Cloud.Issues#809] [Ideas]
### Issue description

We recently had an issue where DNS was causing CAA DNS requests to fail (SERVFAIL) and as a result cloudflare could not issue renewed certificates for some of our hostnames.

The issue was resolved by correcting the DNS server with the client.  

However the umbraco manage hostnames screen was reporting the hostnames as "Protected" even though cloudflare could not issue certificates correctly.

![image](https://github.com/umbraco/Umbraco.Cloud.Issues/assets/5919075/8280baa5-0aff-47c6-bc64-785bc1e64b7a)

This happened even when removing and re-adding the hostnames.

When there is major issue (in this example where the selected hostname does not even point to cloudflare), the dashboard shows the error as follows;

![image](https://github.com/umbraco/Umbraco.Cloud.Issues/assets/5919075/7585b831-8f6c-40af-a39f-8b099fa73a07)

Followed by

<img width="465" alt="image" src="https://github.com/umbraco/Umbraco.Cloud.Issues/assets/5919075/f4835fff-5d5e-428b-9446-ba6a32241c29">

With the detail of the error as follows;

<img width="292" alt="image" src="https://github.com/umbraco/Umbraco.Cloud.Issues/assets/5919075/2f10e3fd-c5a5-4126-8a85-8d0b9b357534">

However when this CAA DNS issue occurred, the status of "Protected" was shown, when the site actually was failing due to a certificate issue.

We request that the dashboard be enhanced to show the actual state of the certificate/protection and not mask any error states.

**Comment by sajumb** [comment]
Thanks for the feature request!
